### DISCUSSION TITLE
List of Hubs components

### DISCUSSION THREAD

**Discussion by mbncr** [mozilla/hubs#4130] [General Questions & Discussion] [answered]
Hi, 
For a few weeks now I've been searching for some form of list of the Hubs components, and use cases. 
In particular, I'm trying make-up for the lack of a comprehensive per-component documentation in the Blender Hubs Exporter plugin.
Are there any resources you'd recommend checking?

Thanks in advance. 😉

**Comment by j-conrad** [comment, marked as answer]
The list of components is evolving slowly with each improvement to the [hubs-blender-exporter](https://github.com/MozillaReality/hubs-blender-exporter) add-on. Help with more thorough documentation would be greatly appreciated. 
Some of those components have some explanation in the Spoke documentation, for example: https://hubs.mozilla.com/docs/physics-and-navigation.html
Others have been featured within tutorial videos such as these: https://www.youtube.com/playlist?list=PLCxaiaRxTL69cYpASBeIJBUtJxoiykhk3

Feel free to ask questions in the Hubs discord as well.

**Reply by mbncr** [reply]
Thank you. That's actually the kind of documentation I want to contribute to, although I'm coming from a bit further on the dev side ;)
I was not sure where to start this on the Hubs Discord, but I'll definitely follow up when I'll have some updates to share.

**Reply by Softvision-GeluHaiduc** [reply]
Marking this as answered, unmark the answer if you need any more information. Thank you!
